test/cloud-slack-dev-e2e/cloud_slack_dev_e2e_test.go

```diff
@@ -380,16 +378,13 @@ func TestCloudSlackE2E(t *testing.T) {
 
 			t.Log("Testing ping for not connected deployment #2")
 			command = "ping"
-			expectedBlockMessage := notConnectedMessage(deployment2.Name, deployment2.ID)
+			expectedMessage = fmt.Sprintf("The cluster %s (id: %s) is not connected.", deployment2.Name, deployment2.ID)
```

Comment:
In my opinion, we don't need to do the exact match here. Those test checks for the overall flow. The more strict approach for checking messages is implemented in the integration tests. WDYT @pkosiec?

I proved that it is passing here: https://github.com/kubeshop/botkube/actions/runs/7836309606

Reply:
Sounds good to me 👍